contracts/modules/v2/StakingPool.sol

```diff
@@ -411,6 +413,15 @@ contract StakingPool is IStakingPool, ERC721 {
     WithdrawParams[] memory params
   ) public returns (uint stakeToWithdraw, uint rewardsToWithdraw) {
 
+    // Prevent withdrawals while the manager is locked in governance
+    {
+      uint managerLockedInGovernanceUntil = nxm.isLockedForMV(manager());
+      require(
+        managerLockedInGovernanceUntil > block.timestamp,
```

Comment:
Shouldn't the operator be `<`? The condition should say `lockedForMV` timestamp to be in the past.